apis/nucleus/src/components/listbox/interactions/keyboard-navigation/keyboard-nav-container.js

```diff
@@ -52,20 +53,12 @@ export default function getListboxContainerKeyboardNavigation({
     }
 
     const container = event.currentTarget.closest('.listbox-container');
-    const inSelection = isModal();
```

Comment:
what modal means in this context?

Reply:
it means is in selection mode:
(or maybe that there is another listbox for the same object that is in selection mode)